Update for scale100

rayer1011 opened the pull request:
Hello Arturo!

I've had some people test this without any issues. They can run the 1P, 3P, and 5P methods smoothly.
Some of them have also tested filtering out certain special cards, and the results are good.
So, I think it's time to make a pull request for it.
However, there are a few images left unchanged from scale125—CountrySelect2 and SelectExpansion—since I couldn't find these images on my end.

NotWeebJack commented:
hi i tried this but it get stuck on platin

NotWeebJack commented:
To make this work with 100 i renamed the scale100 folder to scale125 and inside the script i changed the scale param from 277 to 287

rayer1011 commented:
> To make this work with 100 i renamed the scale100 folder to scale125 and inside the script i changed the scale param from 277 to 287
> 
> We need a selector for 100/125 to make it work easily
> 
> PS thanks for this update <3

check out the discord. there's a community-mods forum. I've made a post there.
